content/updates/2020-12/1-Convert-variant-products-into-simple-products.md

```diff
@@ -0,0 +1,17 @@
+---
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Convert variant products into simple products
+::: meta-data type="New" features="Governance, Productivity" available="late December" link-to-doc="../articles/enrich-products-variants.html#convert-a-variant-product-to-a-simple-one"
```

Comment:
I would change the documentation link: https://help.akeneo.com/pim/serenity/articles/enrich-products-variants.html#convert-a-variant-product-to-a-simple-one

Reply:
It's the same link, Momo. 😉